__tests__/app.test.js

```diff
@@ -156,4 +156,91 @@ describe('GET /api/articles/:article_id/comments', () => {
             expect(body.msg).toBe('Article not found')
         })
     });
+});
+describe('POST /api/articles/:article_id/comments', () => {
+    test('201: should add a comment for an article and respond with the posted comment', () => {
+        const commentToPost = {
```

Comment:
great job on testing, can we add another 201 that it ignores unnecessary properties?